### PR TITLE
Bump corefx

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

File: mcs/class/System/common.sources [hunk 1]
```diff
@@ -872,7 +872,7 @@ corefx/SR.cs
 
 ../../../external/corefx/src/Common/src/System/Net/Logging/NetEventSource.Common.cs
 
-../../../external/corefx/src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs
+#../../../external/corefx/src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs
```
Comment (akoeplinger) [inline mcs/class/System/common.sources]: can be removed instead of commented out I assume?

File: mcs/class/corlib/ReferenceSources/String.cs [hunk 2]
```diff
@@ -41,6 +41,11 @@ namespace System
 {
 	partial class String
 	{
+		public String (ReadOnlySpan<char> value)
+		{
+			throw new NotImplementedException ("string ctor");
```
Comment (akoeplinger) [inline mcs/class/corlib/ReferenceSources/String.cs]: what's our plan for implementing the new Span overloads in code we don't share with corefx? can we grab the implementation from coreclr for this?

Comment (marek-safar) [reply mcs/class/corlib/ReferenceSources/String.cs]: The plan is to switch to CoreFX code where possible. However, in this specific case we need runtime support

Comment (akoeplinger) [reply mcs/class/corlib/ReferenceSources/String.cs]: can we just copy the string out of the span until we have runtime support?

Comment (marek-safar) [reply mcs/class/corlib/ReferenceSources/String.cs]: that's what this method will do, except it's a constructor on a string type so it's not just copy and paste

**Comment by marek-safar** [comment]
@monojenkins Linux x64

**Comment by marek-safar** [comment]
@monojenkins build Linux x64

**Comment by marek-safar** [comment]
@monojenkins build Linux x64

**Comment by marek-safar** [comment]
@monojenkins build Linux x64 mcs

**Comment by marek-safar** [comment]
commit apidiff

**Comment by marek-safar** [comment]
@monojenkins build Linux x64 mcs

**Comment by marek-safar** [comment]
commit apidiff

**Comment by marek-safar** [comment]
@monojenkins build Linux x64 mcs

**Comment by marek-safar** [comment]
@monojenkins build Linux x64 mcs

**Comment by marek-safar** [comment]
@monojenkins build Linux x64 mcs

**Comment by marek-safar** [comment]
@monojenkins build Linux x64 mcs

**Pull request merged**